pyais/messages.py

```diff
@@ -776,7 +776,7 @@ class MessageType6(Payload):
     spare_1 = bit_field(1, bytes, default=b'')
     dac = bit_field(10, int, default=0, signed=False)
     fid = bit_field(6, int, default=0, signed=False)
-    data = bit_field(920, bytes, default=b'', variable_length=True)
```

Comment:
I think that I would prefer to keep this keyword as meta data about the field. Even tough it doesn't change anything practically. But by adding the keyword `variable_length_True` the intention becomes quite clear. Also it might be of use later on.

Reply:
Ok, you're the boss:)